setup.py

```diff
@@ -195,6 +200,11 @@ def run(self, extensions):
 
 if framework in ("all", "jax"):
     dlfw_builder_funcs.append(JaxBuilder)
+    try:
+        import pybind11
+    except ImportError:
+        print("ERROR: pybind11 not present. Can't install JAX backend without it. You can install it with `pip install pybind11`.")
```

Comment:
```suggestion
        print("ERROR: pybind11 not present. You can install it with `pip install pybind11`.")
```
Also, is that the right pythonic way? Shouldn't the error message live in the exception instead of a random print?

Reply:
Yeah I agree, instead of doing this print statement followed by `raise`, a meaningful error with a message must be displayed. In this specific case I'd argue that any exception handling is not needed and python's stack trace for the `ImportError` in case pybind11 is not installed would be sufficient.

Reply:
Good idea. Only doing `import pybind11` now.